client/state/posts/reducer.js

```diff
@@ -415,7 +418,10 @@ export function edits( state = {}, action ) {
 					// Receive a new version of a post object, in most cases returned in the POST
 					// response after a successful save. Removes the edits that have been applied
 					// and leaves only the ones that are not noops.
-					const postEdits = get( memoState, [ post.site_ID, post.ID ] );
+					const postEditsLog = get( memoState, [ post.site_ID, post.ID ] ) || [];
```

Comment:
tiny nit, `[]` default could be given as the 3rd argument here, `get` will return that as the default if the path (2nd argument) isn't found. FFTI though, this is inconsequential.

Reply:
No no, that would actually be a horrible bug :wink: The value in state can be `null` in case when there are no active edits on the post. And `get` only uses the default value for `undefined`, nothing else.

Reply:
Ahh ofc! I should have known there'd be good reason! Thanks for clarifying :D